auth/codes.md

```diff
@@ -0,0 +1,16 @@
+# Codes
+
+## REJECTED
+
+```sh
+case .userRejectedRequest return 14001
```

Comment:
Why can't we inherit common error codes from sign? `userRejectedRequest` is already there, we could create a shared pool of errors and avoid creating the same errors with just different code.

Reply:
I think we should distinguish between sessionRequest and authRequest

Reply:
Why? For peers it doesn't matter. For Relay as well. Auth peer doesn't know about Sign peer.

Reply:
I don't have a strong opinion on it. I just think it is not necessary to create a shared pool of errors just for one error.

Reply:
Yea I agree that for only one error it doesn't make sense. Let's revisit this once more shared error occurs.